configure

```diff
@@ -1101,6 +1112,10 @@ def configure_intl(o):
         pprint.pformat(icu_config, indent=2) + '\n')
   return  # end of configure_intl
 
+def configure_engine(o):
+  engine = options.engine or 'v8'
```

Comment:
@kunalspathak Yes, but the line commented here is in `configure` and not `vcbuild.bat`. It would be better if the or-set in this line were replaced with the very first assignment of `options.engine`, which is what I think @benjamingr intended.